content/applications/services/planning.rst

```diff
@@ -0,0 +1,323 @@
+  field will be displayed.
+- When a schedule is published, the employee will only be notified of open shifts for the roles that
+  are assigned to them.
+- When auto-assigning open shifts or planning sales orders, the employee will only be assigned
+  shifts for the roles assigned to them.
+
+Additionally, when a :guilabel:`Default role` is defined:
+
+- When creating a shift for the employee, the default role will be automatically selected.
+- This role will also have precedence over the other roles of the employee when auto-assigning open
+  shifts or planning sales orders.
+
+.. note::
+   If the Planning roles fields are left empty, there are no restrictions in the shift templates and
+   open shifts shared with the employee. However, it’s not possible to use the **Auto Plan** feature
+   for employee with no roles.
+
+.. _planning/materials:
+
+Materials
+---------
+
+**Materials** are resources that can be assigned shifts and working hours but are not employees.
+For example, a construction company could use Materials to create shifts for shared machines
+(e.g., cranes, forklifts).
+
+Similarly to Employees, Materials can be assigned Roles, Default Role, and Working Time.
+
+.. _planning/templates:
+
+Shift Templates
+---------------
+
+To create a shift template, open any existing shift, then tick :guilabel:`Save as template`.
+
+Alternatively, you can go to :menuselection:`Configuration --> Shift Templates`, then press
+:guilabel:`New`. Fill in the :guilabel:`Start Hour` and :guilabel:`Shift Duration`. The shift’s
+:guilabel:`End Time` is then calculated based on the :guilabel:`Company working hours`, taking into
+account working time as well as breaks.
+
+.. example::
+   The Company Working Hours are Monday to Friday, 8 am to 5 pm, with a break between 12 and 1 pm.
+
+   - Creating a Shift Template with a Start Hour of 9 am and a duration of 8 hours will result in
+     the End Hour being 5 pm, based on the Company Working hours and the 1-hour break.
+   - Creating a Shift Template with a Start Hour of 10 am and a duration of 10 hours will result in
+     the End Hour of 10 am the following day, as the Company is closed at 5 pm according to Company
+     Working Hours.
+
+Additionally, for each shift template, you can also configure:
+
+- :guilabel:`Role` - This shift will be linked to this specific role only.
+- :guilabel:`Project` - Selecting a project allows you to keep track of shifts that are dedicated to
+  work on the project.
+
+.. _planning/shifts:
+
+Planning shifts
+===============
+
+On opening the Planning app, the planning view shows the users their own shifts. Users with admin
+roles can also see :guilabel:`Schedule by Resource`, :guilabel:`Role`, :guilabel:`Project`, or
+:guilabel:`Sales Order`, as well as reporting and configuration menus.
+
+The overview is displayed in the Gantt view, which allows you to edit the shifts with a drag and
+drop, resize, split, and duplicate without having to open the individual shift.
```

Comment:
The overview is displayed in the Gantt view, allowing you to edit the shifts with a drag and drop, resize, split, and duplicate without opening the individual shift.


Not sure I would keep this, it doesn't add guidelines on how to use something so not sure.